examples/simple/src/Layout.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Layout } from 'react-admin';
+import { withStyles, createStyles } from '@material-ui/core/styles';
+
+const styles = createStyles({
+    root: {
+        overflow: 'auto', // Required for expanding nodes on hover when dragging elements in ra-tree
```

Comment:
can you restrict to `overflow-x`? Should we make it the default style in the ra Layout?

Reply:
> Should we make it the default style in the ra Layout?

I'm not sure yet. I'll have to check our demo pages to ensure there are no side effects